src/TaggedInput.jsx

```diff
@@ -144,21 +144,23 @@ var TaggedInput = React.createClass({
   _handleRemoveTag: function (index) {
     var self = this, s = self.state, p = self.props;
 
-    if (p.onBeforeRemoveTag(index)) {
-      var removedItems = s.tags.splice(index, 1);
+    if (!p.onBeforeRemoveTag(index)) {
```

Comment:
I go for early returns too, but only when they help me avoid extra operations further down the line. It wasn't the case here so I think its fine keep it inside the intended if.